### ISSUE TITLE
Codebase contains references to old avisynth.org

### ISSUE THREAD

**Issue by Enyium** [AviSynth/AviSynthPlus#319]
When you [search the codebase](https://github.com/AviSynth/AviSynthPlus/search?q=avisynth.org), there are many references to the old http://www.avisynth.org/ website, including in `avisynth.h`, on which many links give you 404 errors and that still refers you to a legacy Sourceforge repository.

The links should probably be changed to point to http://avisynth.nl/. E.g., http://www.avisynth.org/mediawiki/AviSynth_FAQ becomes http://avisynth.nl/index.php/AviSynth_FAQ.

**Comment by Reel-Deal** [comment]
Most of those instances come from the non-English documentation. I actually wanted to open up an issue to see what pinterf and and qyot27 think about deleting all of the foreign documentation, because all of it is far behind the English docs. The likelihood of someone that speaks a particular language updating the docs is slim to none. Plus, translation tools, while not perfect, they have come a long way.

But back to the original topic. Slowly but surely I will go through all of the English documentation and replace old links with current ones.

**Comment by pinterf** [comment]
Reel-Deal is right, translations are way behind the English version and will never be maintained.
Not only http://www.avisynth.org/ but most of the source code references Avisynth version 2.5.

**Comment by qyot27** [comment]
Indeed, I have thought about the case of just deleting the non-English docs, as even the idea of moving them to a second repo wouldn't accomplish all that much (it likely wouldn't be updated after being moved, so literally nothing changes - anyone wanting to have up-to-date docs for other languages would be vastly better served by working with the current RST/Sphinx docs anyway), and people could still grab the last version by thumbing back through the git history.

Source-code wise, yeah, avisynth.nl would be better in that regard.  The version number references in the header, though, that's probably better left as-is unless we decide to do a full overhaul of all the code formatting and includes and what-all.  It would make sense for AviSynth+-specific sources to be identified as such, rather than just copying the big header block from something as far back as 2.5 or even 1.0.

**Comment by Reel-Deal** [comment]
I'm in total favor of deleting the non-English docs. Moving them to a separate repo is also not a bad idea, at least for "archival" purposes. I have a PR ready to go :)

I guess the same thing can be done with the FilterSDK folder which are not up-to-date. I believe the SDK sections in the rst docs have been updated.

**Comment by Enyium** [comment]
> I guess the same thing can be done with the FilterSDK folder which are not up-to-date. I believe the SDK sections in the rst docs have been updated.

https://avisynthplus.readthedocs.io/en/latest/avisynthdoc/FilterSDK/Cplusplus_api.html contains valuable documentation that I used during development. I also link to it in my doc comments a lot.

**Comment by Reel-Deal** [comment]
>https://avisynthplus.readthedocs.io/en/latest/avisynthdoc/FilterSDK/Cplusplus_api.html contains valuable documentation that I used during development. I also link to it in my doc comments a lot.

That is contained in the rst docs: https://github.com/AviSynth/AviSynthPlus/tree/master/distrib/docs/english/source/avisynthdoc/FilterSDK - this will not be deleted. 

What I was referring to is this: https://github.com/AviSynth/AviSynthPlus/tree/master/distrib/FilterSDK - They are outdated html files.

**Comment by qyot27** [comment]
This has now been resolved in both the docs and the main source tree as of https://github.com/AviSynth/AviSynthPlus/commit/d6baa548656b8bc5e0cdf7b70fbdb04f26310a1c
